### PR TITLE
docs(consumer_group): add delete api for consumer group configuration

### PR DESCRIPTION
**Pull request by yzyyi**
### Summary

We'd like an API to support deleting the current consumer group configurations.

### Reason
[] FT-3564(https://konghq.atlassian.net/browse/FT-3564)

### Testing
Test by using curl -i -X DELETE http://{HOSTNAME}:8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced

### CODE REVIEW

File: src/gateway/kong-enterprise/consumer-groups/index.md [hunk 1]
```diff
@@ -447,6 +447,53 @@ http DELETE :8001/consumers/DianaPrince/consumer_groups/JL
     }
     ```
 
+## Delete consumer group configurations
+
+If you still need the consumer group but wanted to delete the configuations, 
+you can delete it. This removes configurations for the consumer group.
```
Comment (lena-larionova) [inline src/gateway/kong-enterprise/consumer-groups/index.md:453]:
```suggestion
You can also clear the configuration of a consumer group without deleting the consumer group itself.
```

File: src/gateway/kong-enterprise/consumer-groups/index.md [hunk 2]
```diff
@@ -447,6 +447,53 @@ http DELETE :8001/consumers/DianaPrince/consumer_groups/JL
...
+The consumers in the group are not deleted and are still in the consumer group.
```
Comment (lena-larionova) [inline src/gateway/kong-enterprise/consumer-groups/index.md:454]:
```suggestion

With this method, the consumers in the group aren't deleted and are still in the consumer group.
```

File: src/gateway/kong-enterprise/consumer-groups/index.md [hunk 3]
```diff
@@ -447,6 +447,53 @@ http DELETE :8001/consumers/DianaPrince/consumer_groups/JL
...
+    If successful, you receive see the following response:
+    ```
+    HTTP/1.1 204 No Content
+    ```
+    
+1. To verify, check the consumer object configuration:
+
+{{ check_group1 | indent | replace: " </code>", "</code>" }}
+
+    Response, notice the plugins object in the response is gone.
```
Comment (lena-larionova) [inline src/gateway/kong-enterprise/consumer-groups/index.md:484]:
```suggestion
    Response, without a `plugins` object:
```

**Comment by lena-larionova** [comment]
@yzyyi Is this a 3.1 feature? If so, can you please rebase and open this PR against `release/gateway-3.1`?

File: src/gateway/kong-enterprise/consumer-groups/index.md [hunk 4]
```diff
@@ -447,6 +447,53 @@ http DELETE :8001/consumers/DianaPrince/consumer_groups/JL
...
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http DELETE :8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \
+```
+{% endnavtab %}
+{% endnavtabs %}
+{% endcapture %}
+
+{{ delete_consumer_group_config| indent | replace: " </code>", "</code>" }}
```
Comment (lena-larionova) [inline src/gateway/kong-enterprise/consumer-groups/index.md:473]:
```suggestion
{{ delete_consumer_group_config | indent | replace: " </code>", "</code>" }}
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

File: src/gateway/kong-enterprise/consumer-groups/index.md [hunk 5]
```diff
@@ -447,6 +447,53 @@ http DELETE :8001/consumers/DianaPrince/consumer_groups/JL
...
+
+With this method, the consumers in the group aren't deleted and are still in the consumer group.
+
+1. Delete the consumer group configuration using the following request:
+
+{% capture delete_consumer_group_config %}
+{% navtabs codeblock %}
+{% navtab cURL %}
+```bash
+curl -i -X DELETE http://{HOSTNAME}:8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \
```
Comment (lena-larionova) [inline src/gateway/kong-enterprise/consumer-groups/index.md:462]:
```suggestion
curl -i -X DELETE http://{HOSTNAME}:8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced
```

File: src/gateway/kong-enterprise/consumer-groups/index.md [hunk 6]
```diff
@@ -447,6 +447,53 @@ http DELETE :8001/consumers/DianaPrince/consumer_groups/JL
...
+```
+{% endnavtab %}
+{% navtab HTTPie %}
+```bash
+http DELETE :8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced \
```
Comment (lena-larionova) [inline src/gateway/kong-enterprise/consumer-groups/index.md:467]:
```suggestion
http DELETE :8001/consumer_groups/JL/overrides/plugins/rate-limiting-advanced
```

**Review by lena-larionova — APPROVED** [review]
Fixed a couple of minor things (grammar, typos) and some minor rephrasing - applied it in the interest of time. Ready to merge if you're happy with the changes.

**Review by lena-larionova — CHANGES_REQUESTED** [review]
Accidentally approved before PR was moved to 3.1 - please move it so that we can merge.

**Review by yzyyi — COMMENTED** [review]
Thanks, yes we can merge.

**Comment by yzyyi** [comment]
rebased to release/3.1

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**